frontend/src/components/UserDetail/PastRides.tsx

```diff
@@ -53,7 +46,7 @@ const PastRides = ({ isStudent, rides }: pastRideProps) => {
 
       return (
         <tr key={index}>
-          <TableRow values={inputValues} />
+          <Row data={inputValues} colSizes={colSizes} />
```

Comment:
if any of the input values are strings/numbers, you can use it as is, and dont need to use the { data: ... } format